Automattic/jetpack
# Memberships: update the required plan for memberships to be conditional

Make the upgrade plan conditional based on a flag outside of this repo

Fixes #

#### Changes proposed in this Pull Request:

We are about to switch from Pro to Personal as a required plan for premium blocks

#### Jetpack product discussion

This is a wpcom-specific change

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

It has to be tested on wpcom

0. Make sure the local environment remains unchanged
1. Sandbox your test site! The required plan comes from it
2. Apply the backend diff to your sandbox
3. Start a new post and add a Payment Button
4. It should ask you to upgrade to Pro
5. Remove the ENABLE_PRO_PLAN constant from the constants file
6. Reload and add a new payments button
7. It should ask you to upgrade to Personal
8. Click on the upgrade link and make sure the checkout shows Personal
